absorption_demo.ipynb

```diff
@@ -250,24 +198,31 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "6\n",
-      "389125\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "correction = sc.neutron.convert(correction, Dim.Wavelength, Dim.Tof)\n",
+    "correction = sc.neutron.convert(correction, \"Wavelength\", \"Tof\")\n",
```

Comment:
You have to use lower-case here I think, otherwise `neutron.convert` will not recognize this as known/supported dimensions.

Reply:
Of course it should be lowercase. Fixed.